test/OffsetHelper.test.ts

```diff
@@ -249,45 +249,6 @@ describe("OffsetHelper", function () {
   });
 
   describe("#autoOffsetExactOut{ETH,Token}()", function () {
-    it("should retire 1.0 TCO2 using a WETH swap and NCT redemption", async function () {
-      const { offsetHelper, addr2, nct, weth } = await loadFixture(
-        deployOffsetHelperFixture
-      );
-
-      // first we set the initial chain state
-      const wethBalanceBefore = await weth.balanceOf(addr2.address);
-      const nctSupplyBefore = await nct.totalSupply();
-
-      // then we calculate the cost in WETH of retiring 1.0 TCO2
-      const wethCost = await offsetHelper.calculateNeededTokenAmount(
-        addresses.weth,
-        addresses.nct,
-        ONE_ETHER
-      );
-
-      // then we use the autoOffset function to retire 1.0 TCO2 from WETH using NCT
-      await (await weth.approve(offsetHelper.address, wethCost)).wait();
-      await offsetHelper.autoOffsetExactOutToken(
-        addresses.weth,
-        addresses.nct,
-        ONE_ETHER
-      );
-
-      // then we set the chain state after the transaction
-      const wethBalanceAfter = await weth.balanceOf(addr2.address);
-      const nctSupplyAfter = await nct.totalSupply();
-
-      // and we compare chain states
-      expect(
-        formatEther(wethBalanceBefore.sub(wethBalanceAfter)),
-        `User should have spent ${formatEther(wethCost)}} WETH`
-      ).to.equal(formatEther(wethCost));
-      expect(
-        formatEther(nctSupplyBefore.sub(nctSupplyAfter)),
-        "Total supply of NCT should have decreased by 1"
-      ).to.equal("1.0");
-    });
-
     it("should retire using a MATIC swap and NCT redemption", async function () {
```

Comment:
this could also be added within the `TOKEN_POOLS` loop, right?

Reply:
Indeed! I left it out since originally only NCT was tested but I agree that it'd be cleaner to test out both pool tokens whenever we can. Will make these changes then.

Reply:
I figured that's why you did it, and it could have made sense. I just figured, since we're already here, why not?